tests/helper/prism-loader.ts

```diff
@@ -108,18 +110,28 @@ export function createPrismDOM (): PrismDOM<{}> {
 		const g = global as unknown as Record<string, unknown>;
```

Comment:
FYI we actually don't need the typing here, `const g = global` still works for everything